src/inmanta/moduletool.py

```diff
@@ -1132,5 +1132,6 @@ def get_setup_cfg(self, config_in: Optional[configparser.ConfigParser] = None) -
         config["options"]["zip_safe"] = "False"
         config["options"]["include_package_data"] = "True"
         config["options"]["packages"] = "find_namespace:"
+        config["options.packages.find"]["include"] = "inmanta_plugins"
```

Comment:
You're still missing a `config.add_section("options.packages.find")`, see line 1121.